.releaserc

```diff
@@ -5,6 +5,7 @@
     "@semantic-release/npm",
```

Comment:
the plugin ```@semantic-release/npm``` needs to be removed

Reply:
Ok,I fixed all.It's my first time to use semantic-release.Thanks.

Reply:
@abichinger please add a (sign-off)[https://github.com/casbin/pycasbin/pull/95/checks?check_run_id=1503677334].
Thanks very much for helping me a lot. :)